scripts/g.extension/g.extension.py

```diff
@@ -2140,7 +2184,13 @@ def resolve_xmlurl_prefix(url, source=None):
     gscript.debug("resolve_xmlurl_prefix(url={0}, source={1})".format(url, source))
     if source == "official":
         # use pregenerated modules XML file
-        url = "https://grass.osgeo.org/addons/grass%s/" % version[0]
+        repo_branches = get_github_branches()
+        # Define branch to fetch from (latest or current version)
+        version_branch = "grass{}".format(version[0])
+        if version_branch not in repo_branches:
+            version_branch = repo_branches[-1]
+
+        url = "https://grass.osgeo.org/addons/{}/".format(version_branch)
```

Comment:
Same as the other git_github_branches call.

Reply:
Has been removed...